src/status_im/constants.cljs

```diff
@@ -196,3 +196,5 @@
 (def ^:const history-state-changed "history-state-changed")
 (def ^:const web3-send-async "web3-send-async")
 (def ^:const web3-send-async-callback "web3-send-async-callback")
+(def ^:const scan-qr-code "scan-qr-code")
+(def ^:const scan-qr-code-callback "scan-qr-code-callback")
```

Comment:
I don't really see the point of having that here, if there is going to be 2 added constants for each type of event, we could have an `event-type->keyword` function and simply use keywords in the code like `(= type :scan-qr-code-callback)`

Reply:
Also having a 'browser-event' spec listing the keywords would make validation easier

Reply:
There is definitely room for improving this but I'd rather not do it as part of this PR whose goal is to quickly unblock NoBlockNoParty